src/hotspot/share/runtime/arguments.cpp

```diff
@@ -2855,7 +2855,7 @@ jint Arguments::parse_each_vm_init_arg(const JavaVMInitArgs* args, bool* patch_m
       if (jvmciFlag != nullptr && jvmciFlag->is_unlocked()) {
         if (!JVMCIGlobals::enable_jvmci_product_mode(origin, use_graal_jit)) {
           jio_fprintf(defaultStream::error_stream(),
-            "Unable to enable JVMCI in product mode");
```

Comment:
[pre-existing] Unusual indentation.

Reply:
This kind of indentation is pretty common in this file. I'd rather not touch it here.